docs/docs/deploy/configure.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Configure
+sidebar_position: 5
+---
+
+There are two types of configuration in Firezone:
+
+* [Runtime configuration](#runtime-configuration): Application configuration
+  related to day-to-day operation of Firezone.
+* [Deployment configuration](#deployment-configuration): Deployment or
+  infrastructure-related configuration releveant to running Firezone on-prem.
+
+## Runtime Configuration
+
+Most day-to-day configuration of Firezone can be done via the Web UI or
+REST API. See our [REST API documentation](../../reference/REST API) for more
```

Comment:
Just flagging this line in case the API is not merged this update. Also there's a space in the path so it's not a valid link.

Reply:
Thanks, good catch